packages/xo-web/src/index.scss

```diff
@@ -213,6 +213,12 @@ $select-input-height: 40px; // Bootstrap input height
   border: 1px solid black;
   margin: 5px 10px;
   width: 250px;
+  // "input-group > form-control" and "input-group > input-group-btn" have hidden the notification.
+  // (https://github.com/vatesfr/xen-orchestra/issues/3874)
+  // In bootstrap, The "z-index: 3" is for focused "input-group form-control".
+  // (https://github.com/twbs/bootstrap/blob/master/scss/_input-group.scss#L30TOL34)
```

Comment:
This is not the version of Bootstrap we are using, please check in `package.json`.